_config.yml

```diff
@@ -16,20 +16,23 @@
 #  https://github.com/carpentries/training-template
 carpentry: "swc"
 
-# This option is currently only needed for Data Carpentry workshops.
-# The value of the variable will be ignored for Software Carpentry and
-# Library Carpentry workshops.
+# This option is currently only needed for
+# Data Carpentry and Software Carpentry workshops.
+# The value of the variable will be ignored for Library Carpentry
+# workshops.
 # Possible values are:
 # dc-ecology: for Data Carpentry Ecology workshops
 # dc-genomics: for Data Carpentry Genomics
 # dc-socsci: for Data Carpentry Social Sciences
 # dc-geospatial: for Data Carpentry Geospatial
+# swc-inflammation: for Software Carpentry based on the inflammation dataset
+# swc-gapminder: for Software Carpentry based on the Gapminder dataset
 curriculum: "FIXME"
 
 # If there is both an R and python official version of a curriculum, change the
```

Comment:
Does official here mean versions that are past alpha? if so, we should probably update this to be more specific.

Reply:
Thanks Sarah, I'm adding links to the SWC and DC lesson page that lists our official lessons.